metadata.json

```diff
@@ -52,7 +52,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 3.2.0 < 5.0.0"
+      "version_requirement": ">= 4.0.0 < 5.0.0"
```

Comment:
We use 4.6.1 in theforeman and copied that from voxpupuli and puppetlabs. It's considered the first 4 version you can really rely on.